lms/envs/common.py

```diff
@@ -1042,6 +1045,11 @@
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/modules/**/*.js'))
 )
 
+# Include JS files needed for courseware search
+courseware_js += (
+    'js/search/main.js',
```

Comment:
Just add this to where `courseware_js` is defined directly above, rather than defining the variable and then immediately modifying it.